app/controllers/api/measurement_sessions_controller.rb

```diff
@@ -27,17 +27,6 @@ class MeasurementSessionsController < BaseController
 
     respond_to :json
 
-    def index
-      data = decoded_query_data(params[:q])
-      INT_Q_ATTRS.each { |key| data[key] = data[key].to_i if data.key?(key) }
```

Comment:
You can also delete the `INT_Q_ATTRS ` array on line 21.

Reply:
Good catch!